packages/components/src/focal-point-picker/index.tsx

```diff
@@ -46,36 +97,41 @@ export default function FocalPointPicker( {
 		x: 0.5,
 		y: 0.5,
 	},
-} ) {
+	...restProps
+}: WordPressComponentProps< FocalPointPickerProps, 'div', false > ) {
 	const [ point, setPoint ] = useState( valueProp );
 	const [ showGridOverlay, setShowGridOverlay ] = useState( false );
 
 	const { startDrag, endDrag, isDragging } = useDragging( {
 		onDragStart: ( event ) => {
-			dragAreaRef.current.focus();
+			dragAreaRef.current?.focus();
 			const value = getValueWithinDragArea( event );
+			if ( ! value ) return;
 			onDragStart?.( value, event );
 			setPoint( value );
 		},
 		onDragMove: ( event ) => {
 			// Prevents text-selection when dragging.
 			event.preventDefault();
 			const value = getValueWithinDragArea( event );
+			if ( ! value ) return;
 			onDrag?.( value, event );
 			setPoint( value );
 		},
-		onDragEnd: ( event ) => {
-			onDragEnd?.( event );
+		onDragEnd: () => {
+			onDragEnd?.();
```

Comment:
This never received any event arguments because `endDrag()` is called in an `onBlur`, and the event wasn't passed through. I guess we can keep it as is, because it would be kind of weird for an `onDragEnd` handler to receive a blur event.

Reply:
Good to know, thanks!